src/app/routes/onDemandAudio/getInitialData/index.test.js

```diff
@@ -75,7 +75,9 @@ describe('Get initial data for on demand radio', () => {
       {
         linkText: 'RSS',
         linkUrl: 'https://podcasts.files.bbci.co.uk/p02pc9qc.rss',
+        linkType: 'rss',
       },
+      [],
```

Comment:
Why is there an empty array here?

Reply:
fixed within this PR: https://github.com/bbc/simorgh/pull/10544